src/store/dicom-web/dicom-web-store.ts

```diff
@@ -43,10 +45,17 @@ async function getAllPatients(host: string): Promise<PatientInfo[]> {
  * Collect DICOM data from DICOMWeb
  */
 export const useDicomWebStore = defineStore('dicom-web', () => {
-  const host = process.env.VUE_APP_DICOM_WEB_URL
-    ? ref(process.env.VUE_APP_DICOM_WEB_URL)
-    : useLocalStorage<string>('dicomWebHost', '');
-  const isConfigured = computed(() => !!host.value);
+  const urlParams = vtkURLExtract.extractURLParameters() as UrlParams;
+  const dicomWebFromURLParam = urlParams.dicomweb as string | undefined;
```

Comment:
I'd recommend moving `dicomweb` into a constant. It makes it more obvious if we need to ever configure it down the road.

Reply:
Done.